src/main/java/org/alliancegenome/curation_api/model/entities/bulkloads/BulkLoadFile.java

```diff
@@ -21,6 +30,11 @@
 @AGRCurationSchemaVersion("1.2.4")
 public class BulkLoadFile extends GeneratedAuditedObject {
 	
+	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
+	@KeywordField(name = "dateLastLoaded_keyword", sortable = Sortable.YES, searchable = Searchable.YES, aggregable = Aggregable.YES, valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
+	@JsonView(View.FieldsOnly.class)
+	private OffsetDateTime dateLastLoaded;
```

Comment:
@markquintontulloch we need to add this to LinkML?

Reply:
yeah